data/data-pipeline/data_pipeline/etl/score/etl_score_geo.py

```diff
@@ -27,6 +28,10 @@ def __init__(self, data_source: str = None):
         self.SCORE_LOW_GEOJSON = self.SCORE_GEOJSON_PATH / "usa-low.json"
         self.SCORE_HIGH_GEOJSON = self.SCORE_GEOJSON_PATH / "usa-high.json"
 
+        self.SCORE_SHP_PATH = self.DATA_PATH / "score" / "shapefile"
```

Comment:
This is new. I tried to mirror the existing code, but let me know if you think this needs its own script.

Reply:
I like it!